dali/operators/image/resize/experimental/resize_op_impl_cvcuda.h

```diff
@@ -146,36 +156,43 @@ class ResizeOpImplCvCuda : public ResizeBase<GPUBackend>::Impl {
 
   void RunResize(Workspace &ws, TensorList<GPUBackend> &output,
                  const TensorList<GPUBackend> &input) override {
-    TensorList<GPUBackend> in_frames;
-    in_frames.ShareData(input);
-    in_frames.Resize(in_shape_);
-    PrepareInput(in_frames);
-
-    TensorList<GPUBackend> out_frames;
-    out_frames.ShareData(output);
-    out_frames.Resize(out_shape_);
-    PrepareOutput(out_frames);
-
-
     kernels::DynamicScratchpad scratchpad({}, AccessOrder(ws.stream()));
+    auto allocator = nvcvop::GetScratchpadAllocator(scratchpad);
 
-    auto workspace_mem = op_workspace_.Allocate(workspace_reqs_, scratchpad);
+    auto workspace_mem = AllocateWorkspaces(scratchpad);
 
     for (size_t b = 0; b < minibatches_.size(); b++) {
       MiniBatch &mb = minibatches_[b];
-      resize_op_(ws.stream(), workspace_mem, mb.input, mb.output, mb.min_interpolation,
+      auto reqs = nvcv::TensorBatch::CalcRequirements(mb.count);
+      auto mb_output = nvcv::TensorBatch(reqs, allocator);
+      auto mb_input = nvcv::TensorBatch(reqs, allocator);
+      nvcvop::PushFramesToBatch(mb_input, input, first_spatial_dim_, mb.sample_offset,
+                                mb.frame_offset, mb.count, sample_layout_);
+      nvcvop::PushFramesToBatch(mb_output, output, first_spatial_dim_, mb.sample_offset,
+                                mb.frame_offset, mb.count, sample_layout_);
```

Comment:
This is a bug. Both inputs and outputs should be inserted in one go and skipping the empty samples should be based solely on the output size. The user may request resizing a non-empty tensor to (0, 0), which is not an error AFAIR. Resizing an empty input to non-empty shape _is_ an error and should be raised earlier.